lang/pl.json

```diff
@@ -5,6 +5,7 @@
 			"2": "Wesprzyj mnie!",
 			"3": "Pomóż autorowi"
 		},
+		"pick-a-location": "Wybierz lokalizację!",
```

Comment:
"Wybierz położenie!", może "lokację".
Lokalizacja brzmi jakbyś wybierał język...

Reply:
Nigdy jednoznacznie nie wiem, co ma autor na myśli. Poprawię to i wyślę ponownie. Dzięki za uwagi!

Reply:
Jak coś, jest w tym samym pull requeście @MrVauxs